log4j-core/src/main/java/org/apache/logging/log4j/core/config/Configuration.java

```diff
@@ -179,6 +180,16 @@ default StrSubstitutor getConfigurationStrSubstitutor() {
      */
     AsyncLoggerConfigDelegate getAsyncLoggerConfigDelegate();
 
+    /**
+     * Returns the {@code AsyncWaitStrategyFactory} defined in this Configuration;
```

Comment:
I like Javadocs for getters to say "Gets..."

Reply:
My impression is that the convention used in the JDK is to start javadoc comments with "Returns..." for methods whose main function is to return a value. So I would prefer to the current phrasing.

Reply:
There is no such convention in the JDK, for example in System, we see "Gets" for https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#getSecurityManager-- and also "Determines" and "Returns". I just settled on "Gets".